articles/policies/rate-limits-mgmt-api.md

```diff
@@ -0,0 +1,89 @@
+:::
+
+The rate limits for this API differ depending on whether your tenant is free or paid, production or not.
+
+| Tenant Type | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - |
+| Free or Trial | 10 | 120 |
+| Developer or Developer Pro (created before May 19, 2020) | 50 | 1000 |
+| Enterprise (Production) | 50 | 1000 |
+| Enterprise (Non-production) | 10 | 120 |
+
+Each endpoint is configured with a bucket that defines:
+
+-  Request limit
+-  Rate limit window (per second, per minute, per day, etc.)
+
+```text
+bucket:
+    size: x
+    per_minute: y
+```
+
+For example, the above states that, for the given bucket, there is a maximum request limit of `x` per minute, and for each minute that elapses, permissions for `y` requests are added back. In other words, for each `60 / y` seconds, one additional request is added to the bucket. This occurs automatically until the bucket contains the maximum permitted number of requests.
+
+For some API endpoints, the rate limits are defined per bucket, so the origins of the call do not influence the rate limit changes. For other buckets, the rate limits are defined using different keys, so the originating IP address is considered when counting the number of received API calls.
+
+::: note
+If you are using an API endpoint **not** listed below and you receive rate limit headers as part of your response, see [Anomaly Detection](/anomaly-detection) for more information.
+:::
+
+The aforementioned rate limits include calls made via [Rules](/rules) and are set **by tenant** and not by endpoint.
+
+The following Auth0 Management API endpoints return rate limit-related headers. For additional information about these endpoints, please consult the [Management API explorer](/api/management/v2).
+
+| Endpoint | Path | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - | - |
+| Read users | `GET /api/v2/users` | 40 | 500 |
+| Write users | `POST /api/v2/users` | 20 | 200 |
+| Read logs | `GET /api/v2/logs` | 10 | 100 |
+| Write logs | `GET /api/v2/user/{id}/logs` | 10 | 200|
+| Read clients | `GET /api/v2/clients` | 5 | 100 |
+| Write clients | `POST /api/v2/clients/{id}` | 5 | 100 |
+| Read connections | `GET /api/v2/connections` | 10 | 100 |
+| Write connections | `POST /api/v2/connections/{id}` | 10 | 100 |
+| Write device credentials | `POST /api/v2/device-credentials` | 5 | 100 | 
+| Delete device credentials | `DELETE /api/v2/device-credentials/{id}` | 5 | 100 |
```

Comment:
Only applies for `Developer` and `Developer Pro` subscriptions, please create a separate section.
The remaining rate limits applies for all subscriptions.